### ISSUE TITLE
Hide Tabs Toolsbar

### ISSUE THREAD

**Issue by stephenjmcmahon** [MrOtherGuy/firefox-csshacks#4]
https://github.com/MrOtherGuy/firefox-csshacks/blob/master/chrome/hide_tabs_toolbar.css

Anyone having an issue with not being able to drag the browser around anymore? Removed the code from userChrome.css to show the tabs and was able to drag it around again

Win10
Firefox 66.0b5 (64-bit)

**Comment by MrOtherGuy** [comment]
Sorry I have somehow completely missed this issue.  I take it you mean you lost the ability to drag the window from the navigation toolbar left side (which does happen). That stylsheet does not really support having menubar enabled, it can be toggled with Alt-key and then its drawn over the navbar. If menubar is enabled then the toolbar just wont bee visible.

I do have a fix for the left side of navbar being unusable for window dragging though.

**Comment by doychi** [comment]
@MrOtherGuy, yes. I have the same issue where i cannot drag the whole Firefox window. I have to resize to move the window around
